tests/web/test_info.py

```diff
@@ -31,5 +31,6 @@ def test_api_v1_endpoint(client):
             'self': 'http://localhost/api/v1/',
             'statuses': 'http://localhost/api/v1/statuses/',
             'transactions': 'http://localhost/api/v1/transactions/',
+            'streams_v1': 'ws://localhost:9985/api/v1/streams/valid_tx',
```

Comment:
*There is a bit of personal interpretation in the following statement.*

The difference is that `http://localhost/api/v1/transactions/` represents a "directory". Under it you can find transactions that are the "files".

On the other side `ws://localhost:9985/api/v1/streams/valid_tx` is a "file" since it's the stream endpoint.

We could argue if `streams_v1` should point to `ws://localhost:9985/api/v1/streams/valid_tx` or `http://localhost:9985/api/v1/streams/`